rpi_backlight.py

```diff
@@ -47,34 +48,37 @@ def _set_value(name: str, value: Any) -> None:
 
 def get_actual_brightness() -> int:
     """Return the actual display brightness."""
-    return int(_get_value("actual_brightness"))
+    return int(_get_value("tinker_mcu_bl"))
 
 
 def get_max_brightness() -> int:
     """Return the maximum display brightness."""
-    return int(_get_value("max_brightness"))
+    return 255
 
 
 def get_power() -> bool:
     """Return wether the display is powered on or not."""
     # 0 is on, 1 is off
-    return not int(_get_value("bl_power"))
+    if int(_get_value("tinker_mcu_bl")) == 0:
+        return "Off"
+    else:
+        return "On"
```

Comment:
This function returns a `bool`, not `str`. See return type annotation.

Reply:
Fixed